### PR TITLE
Tenant definition

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**

### CODE REVIEW

File: users/src/main/proto/spine/users/organization.proto [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    //
+    // Unlike the `primary_domain` this field contains full URL, which may lead to a page with information
+    // about the organization. For example: `https://www.teamdev.com/contactus`
+    //
+    net.Url url = 5;
+
+    // A contact email address, if available.
+    net.EmailAddress email = 6;
+
+    // Location information, if available
```
Comment (armiol) [inline users/src/main/proto/spine/users/organization.proto]: Dot is missing.

File: users/src/main/proto/spine/users/organization.proto [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    //
+    net.InternetDomain primary_domain = 3;
+
+    // The URL of the organization's logo picture, if available.
+    net.Url logo_url = 4;
+
+    // The URL of the organization's site or another URL that represents the organization, if available.
+    //
+    // Unlike the `primary_domain` this field contains full URL, which may lead to a page with information
+    // about the organization. For example: `https://www.teamdev.com/contactus`
```
Comment (armiol) [inline users/src/main/proto/spine/users/organization.proto]: Dot is missing.

File: users/src/main/proto/spine/users/tenant.proto [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    TenantId id = 1;
+
+    oneof kind {
+        // The tenant is an organization.
+        OrganizationProfile organization = 2;
+
+        // The tenant is a person.
+        UserProfile person = 3;
+    }
+}
```
Comment (armiol) [inline users/src/main/proto/spine/users/tenant.proto]: Newline is missing.

**Comment by armiol** [comment]
@alexander-yevsyukov please see my comments.

File: users/src/main/proto/spine/users/organization.proto (hunk 2 above)
Comment (alexander-yevsyukov) [inline users/src/main/proto/spine/users/organization.proto]: Fixed.

File: users/src/main/proto/spine/users/organization.proto (hunk 1 above)
Comment (alexander-yevsyukov) [inline users/src/main/proto/spine/users/organization.proto]: Fixed.

File: users/src/main/proto/spine/users/tenant.proto (hunk 3 above)
Comment (alexander-yevsyukov) [inline users/src/main/proto/spine/users/tenant.proto]: Fixed. Posted hint on using IDEA setting to solve this problem once and for all.

**Comment by armiol** [comment]
@alexander-yevsyukov LGTM.

**Pull request merged**